app/components/preprint-status-banner/template.hbs

```diff
@@ -21,7 +21,7 @@
             {{/if}}
         </div>
         <div class="dropdown reviewer-feedback">
-            {{#bs-dropdown closeOnMenuClick=false as |dd|}}
+            {{#bs-dropdown onHide=(action 'closeFeedback') closeOnMenuClick=false as |dd|}}
```

Comment:
`closeFeedback` doesn't do much -- could this be `onHide=(action setUserEnteredReview userActivity)`?